content/automate/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/check_service_retired.rb

```diff
@@ -16,6 +16,13 @@ def main
 
               private
 
+              def log_task_info(task)
+                @handle.log('info', "Service Retire Task:<#{task.id}> <#{task.description}> is not retired, setting retry.")
```

Comment:
Minor. Replace `:` with a space.

`@handle.log('info', "Service Retire Task <#{task.id}> <#{task.description}> is not retired, setting retry.")
`